lib/ansible/modules/cloud/vmware/vmware_content_library_info.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Ansible Project
+# Copyright: (c) 2019, Pavan Bidkar <pbidkar@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_content_library_info
+short_description: Gather information about VMWare Content Library
+description:
+- Module to list the content libraries.
+- Module to get facts about specific content library
+- Content Library feature is introduced in vSphere 6.0 version, so this module is not supported in the earlier versions of vSphere.
+- All variables and VMware object names are case sensitive.
+version_added: '2.9'
+author:
+- Pavan Bidkar (@pgbidkar)
+notes:
+- Tested on vSphere 6.5, 6.7
+requirements:
+- python >= 2.6
+- PyVmomi
+- vSphere Automation SDK
+options:
+    library_id:
+      description:
+      - content library id for which details needs to be fetched.
+      type: str
+      required: False
+extends_documentation_fragment: vmware_rest_client.documentation
+'''
+
+EXAMPLES = r'''
+- name: Get List of Content Libraries
+  vmware_content_library_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+  delegate_to: localhost
+
+- name: Get facts about content library
+  vmware_content_library_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    library_id: '13b0f060-f4d3-4f84-b61f-0fe1b0c0a5a8'
+    validate_certs: no
+  delegate_to: localhost
+'''
+
+RETURN = r'''
+content_lib_details:
+  description: dictionary of content library metadata
+  returned: on success
+  type: dict
+  sample: {
+            "demo-local-lib": {
+            "lib_creation_time": "2019-07-02T11:50:52.242000",
+            "lib_description": "new description",
+            "lib_id": "13b0f060-f4d3-4f84-b61f-0fe1b0c0a5a8",
+            "lib_name": "demo-local-lib",
+            "lib_publish_info": {
+                "authentication_method": "NONE",
+                "persist_json_enabled": false,
+                "publish_url": null,
+                "published": false,
+                "user_name": null
+                },
+                "lib_server_guid": "0fd5813b-aac7-4b92-9fb7-f18f16565613",
+                "lib_type": "LOCAL",
```

Comment:
In the other PR ( https://github.com/ansible/ansible/pull/58716 ), `lib_type` is called `library_type` and the value is either `local` or `subscribed`. Could you use the same in both PR?

Reply:
Did this change. Changed all lib to library to consistent with other content library module

Reply:
Thanks!